front-end/src/App.tsx

```diff
@@ -19,15 +17,13 @@ export default function App() {
       <Route path="/" element={<LogIn/>} />
       <Route path="/login" element={<LogIn/>} />
       <Route path="/signup" element={<SignUp/>} />
-      <Route path="/posts" element={<Posts/>} />
-      <Route path="/posts/new" element={<NewPost/>} />
-      <Route path="/posts/:id" element={<Post />} />
       <Route path="/verify/:token" element={<VerifyEmail />} />
-      <Route path="/posts/:id/edit" element={<EditPost />} />
-      <Route path="/posts/:post_id/comments" element={<Comments />} />
-      <Route path="/posts/:post_id/comments/:id/edit" element={<EditComment />} />
-      <Route path="/userhome" element={<UserHome/>} />
-      <Route path="/logout" element={<LogOut/>} />
+      <Route path="/myprofile" element={<MyProfile/>} />
```

Comment:
We can seperate all the routes to a file called routes.ts then after that we can import that here and loop that specific file.

let say we have a 100 routes. do we need to keep adding a new line for each routes. so for dynamic follow what I suggested.